modin/pandas/test/test_general.py

```diff
@@ -770,7 +779,9 @@ def test_to_pandas_indices(data):
         assert md_df.axes[axis].equals(
             pd_df.axes[axis]
         ), f"Indices at axis {axis} are different!"
-        assert md_df.axes[axis].equal_levels(
+        assert not hasattr(md_df.axes[axis], "equal_levels") or md_df.axes[
```

Comment:
Why?

Reply:
In case of an empty frame, the index is of type RangeIndex and the test fails with AttributeError: 'RangeIndex' object has no attribute 'equal_levels'. This is a bug and it's fixed in #5567. In order not to clutter this PR, I've not included the fixes from #5567, but just added this condition to the test. When #5567 is merged, it could be removed.

Reply:
ok